remove experimental for batch_predict and deprecate the async batch i… (#9768)
opensearch-project/documentation-website: _ml-commons-plugin/remote-models/async-batch-ingestion.md

# Asynchronous batch ingestion
**Deprecated 3.0**
{: .label .label-red }

> This feature is deprecated. For similar functionality, use [Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/). If you'd like to see this feature reinstated, [open an issue](https://github.com/opensearch-project/ml-commons/issues) in the ML Commons repository.

Reviewer: ```suggestion
This feature is deprecated. For similar functionality, use [Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/). If you'd like to see this feature reinstated, [open an issue](https://github.com/opensearch-project/ml-commons/issues) in the ML Commons repository.
{: .warning}

```

# Asynchronous batch ingestion
**Deprecated 3.0**
{: .label .label-red }

> This feature is deprecated. For similar functionality, use [Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/). If you'd like to see this feature reinstated, [open an issue](https://github.com/opensearch-project/ml-commons/issues) in the ML Commons repository.

Reviewer 2: ```suggestion
This feature is deprecated. For similar functionality, use [OpenSearch Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/). If you'd like to see this feature reinstated, [create an issue](https://github.com/opensearch-project/ml-commons/issues) in the ML Commons repository.
```

# Asynchronous batch ingestion
**Deprecated 3.0**
{: .label .label-red }

This feature is deprecated. For similar functionality, use [Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/). If you'd like to see this feature reinstated, [open an issue](https://github.com/opensearch-project/ml-commons/issues) in the ML Commons repository.
{: .warning}

> {: .warning}

Reviewer: ```suggestion
```

# Asynchronous batch ingestion
**Deprecated 3.0**

> {: .label .label-purple }

Reviewer: ```suggestion
{: .label .label-red }
```

> This feature has been deprecated. Please use DataPrepper for similar functionality. To request reinstatement of this feature, open an issue at 

Reviewer: ```suggestion
This feature is deprecated. For similar functionality, use [Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/). If you'd like to see this feature reinstated, [open an issue](https://github.com/opensearch-project/ml-commons/issues) in the ML Commons repository.
```

> https://github.com/opensearch-project/ml-commons/issues 

Reviewer: ```suggestion
```
